src/Database/Statement/BufferedStatement.php

```diff
@@ -186,18 +186,15 @@ public function bind(array $params, array $types): void
     /**
      * @inheritDoc
      */
-    public function lastInsertId(?string $table = null, ?string $column = null)
+    public function lastInsertId(?string $table = null, ?string $column = null): string|int
     {
         return $this->statement->lastInsertId($table, $column);
     }
 
     /**
-     * {@inheritDoc}
-     *
-     * @param string|int $type The type to fetch.
-     * @return array|false
+     * @inheritDoc
      */
-    public function fetch($type = self::FETCH_TYPE_NUM)
+    public function fetch(string|int $type = self::FETCH_TYPE_NUM): stdClass|array|false
```

Comment:
This can return `stdClass` which was undocumented. That is causing all of the psalm errors.

Reply:
Solving all of those type errors is going to be tedious :grimacing:

Reply:
Shouldn't the return type be just `mixed` to match `StatementInterface::fetch()`?

Reply:
I changed it to return `mixed`. We don't support the PDOStatement fetch type that returns true, but specifying `stdClass|array|false` causes too many type errors.